index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"/>
+</head>
+<body>
+    <div class="main_box">
+        
+        <div class="btn_one">
+          <label for="check">
+            <i class="fas fa-bars"  id="me"></i>    
+          </label>
+        </div>  
+        <div class="sidebar_menu">
+          <div class="logo"   id="you">
+            <a href="#">MYlogo</a>
+            </div>
+            <ul class="fa-ul">
+                <li><span class="fa-li"><a href="#"><i class="fas fa-house-user"></i></span>
+                    Home</a></i>
+                    
+                <li><span class="fa-li"><a href="#"><i class="fab fa-avianex"></i></span>
+                    Explore</a></i>
+                <li><span class="fa-li"><a href="#"><i class="fab fa-bity"></i></span>
+                    Shorts</a></i>
+                <li><span class="fa-li"><a href="#"><i class="fab fa-bity"></i></span>
+                    Subsriptions</a></i>
+                <li><span class="fa-li"><a href="#"><i class="fa fa-folder"></i></span>
+                    Library</a></i>
+                <li><span class="fa-li"><a href="#"><i class="fab fa-bity"></i></span>
+                    History</a></i>
+                <li><span class="fa-li"><a href="#"><i class="fab fa-bity"></i></span>
+                    Your videos</a></i>
+                <li><span class="fa-li"><a href="#"><i class="fa fa-film"></i></span>
+                    Watch later</a></i>
+                <li><span class="fa-li"><a href="#"><i class="fab fa-bity"></i></span>
+                    Liked videos</a></i>
+            </ul>
+        </div>     
+    </div>
+    <div class="nav-bar">
+        <div class="nav">
```

Comment:
I think it would be appropriate if the nav tag was used instead of the div tag.

Reply:
- Good job suggesting this change, please I'd advise you always give reasons for suggesting a change, this will help the author of the project understand better why he needs to make the change. 😄 😄